.pipelines/trigger.yaml

```diff
@@ -0,0 +1,51 @@
+pr:
+  branches:
+    include:
+      - master
+      - release/*
+  paths:
+    exclude:
+      - ".devcontainer"
+      - ".hooks"
+      - ".vscode"
+      - ".github"
+      - docs
+
+
+trigger:
+  branches:
+    include:
+      - gh-readonly-queue/master/*
+  tags:
+    include:
+      - "*"
+
+
+resources:
+  repositories:
+  - repository: azure-container-networking
+    type: github
+    name: Azure/azure-container-networking
+    endpoint: 'Azure-ACN RO Service Connection'
+    ref: refs/heads/feature/ob-onboard-0
```

Comment:
Does this need to be changed before we merge this to master?

Reply:
Fine for now until we are past the validation/master parity stage. Then we'll update this ref to master.